include/ensmallen_bits/problems/schaffer_function_n1.hpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file schaffer_function_n1.hpp
+ * @author Sayan Goswami
+ *
+ * Implementation of Schaffer function N.1.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef ENSMALLEN_PROBLEMS_SCHAFFER_FUNCTION_N1_HPP
+#define ENSMALLEN_PROBLEMS_SCHAFFER_FUNCTION_N1_HPP
+
+namespace ens {
+namespace test {
+
+template<typename MatType = arma::mat>
+class SchafferFunctionN1
+{
+ public:
+  SchafferFunctionN1() : numObjectives(2), numVariables(1)
+  {/* Nothing to do here. */}
+
+  arma::vec Evaluate(const MatType& coords) {
```

Comment:
That brace needs to be on the next line. There are a couple other places where this needs to be done, take a look :).

Reply:
Yes sure!